consensus/types/src/chain_spec.rs

```diff
@@ -468,6 +475,12 @@ impl ChainSpec {
             .is_some_and(|fulu_fork_epoch| fulu_fork_epoch != self.far_future_epoch)
     }
 
+    /// Returns true if `GLOAS_FORK_EPOCH` is set and is not set to `FAR_FUTURE_EPOCH`.
+    pub fn is_gloas_scheduled(&self) -> bool {
```

Comment:
added this to resolve build issues since the function was referenced but didn't exist